README.md

```diff
@@ -49,7 +42,14 @@ To get a local copy up and running follow these simple example steps.
     cd space-hub
 
 ### Install
-- Install the dependencies by running `npm i` in your terminal
+- To Install this project on your local enviroment you just need to have install `npm` on your local machine.
```

Comment:
- [ ] [OPTIONAL] Great job adding npm instructions, but your instructions could more detailed as below:


## EXAMPLE
```
### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/kidde60/covid-19-updates/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/kidde60/covid-19-updates.git
```
```
- Once the files are on your machine, open the _covid-19-updates_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up

## 🛠 Installation & Set Up

1. Install Node Modules

   ```sh
   npm install
   ```
```
2. Start the development server

   ```sh
   npm start
   ```
```
3. Open [http://localhost:3000](http://localhost:3000) to view it in the browser.

## 🚀 Building and Running for Production

1. Generate a full static production build

   ```sh
   npm run build
   ```